Formula/at-spi2-core.rb

```diff
@@ -24,13 +23,9 @@ class AtSpi2Core < Formula
   depends_on "xorgproto"
 
   def install
-    ENV.refurbish_args
```

Comment:
Not sure on reason for this, so trying out without it.

Reply:
Yeah if it's not needed, let's remove it.